bridge/deploy/07_deploy_federationProxy.js

```diff
@@ -1,14 +1,16 @@
+const address = require('../hardhat/helper/address');
 
-module.exports = async function({getNamedAccounts, deployments, network}) { // HardhatRuntimeEnvironment
-  const {deployer, multiSig, proxyAdmin, bridgeProxy, federatorProxy} = await getNamedAccounts();
+module.exports = async function(hre) { // HardhatRuntimeEnvironment
+  const {getNamedAccounts, deployments, network} = hre;
+  const {deployer, federatorProxy} = await getNamedAccounts();
   const {deploy, log} = deployments;
 
   if (federatorProxy) return;
 
   const FederationV2 = await deployments.get('FederationV2');
-  const proxyAdminAddress = proxyAdmin ?? (await deployments.get('ProxyAdmin')).address;
-  const multiSigAddress = multiSig ?? (await deployments.get('MultiSigWallet')).address;
-  const bridgeProxyAddress = bridgeProxy ?? (await deployments.get('BridgeProxy')).address;
+  const proxyAdminAddress = await address.getProxyAdminAddress(hre);
+  const multiSigAddress = await address.getMultiSigAddress(hre);
```

Comment:
we should adapt every deploy script to use this whenever possible, right? for example in 05_deploy_bridgeProxy.js we're still doing it in the old way
if we won't do it on this PR, I'd maybe create and GH issue + JIRA ticket to not lose sight of it

Reply:
yes we should update it 👍

Reply:
Sure, the PR was getting too big, but definitely, I can do it in this one

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4